it/controllers/pensions/paymentsIntoPensions/ReliefAtSourcePensionsControllerISpec.scala

```diff
@@ -19,270 +19,59 @@ package controllers.pensions.paymentsIntoPensions
-    val expectedCheckProviderText = "You can check with your pension provider whether this applies to you."
-  }
-
-  object ExpectedAgentEN extends SpecificExpectedResults {
-    val expectedH2 = "Did your client pay into a RAS pension?"
-    val expectedError = "Select yes if your client paid into a RAS pension"
-    val expectedParagraph = "These are pensions your client pays into from:"
-    val expectedExample1 = "your client’s net income (after tax has been deducted), if they’re employed"
-    val expectedExample2 = "your client’s taxable income, if they’re self-employed"
-    val expectedPensionProviderText = "Your pension provider then claims tax relief for your client."
-    val expectedCheckProviderText = "You can check with your pension provider whether this applies to your client."
-  }
-
-  object ExpectedAgentCY extends SpecificExpectedResults {
-    val expectedH2 = "Did your client pay into a RAS pension?"
-    val expectedError = "Select yes if your client paid into a RAS pension"
-    val expectedParagraph = "These are pensions your client pays into from:"
-    val expectedExample1 = "your client’s net income (after tax has been deducted), if they’re employed"
-    val expectedExample2 = "your client’s taxable income, if they’re self-employed"
-    val expectedPensionProviderText = "Your pension provider then claims tax relief for your client."
-    val expectedCheckProviderText = "You can check with your pension provider whether this applies to your client."
-  }
-
-  object CommonExpectedEN extends CommonExpectedResults {
-    val expectedCaption: Int => String = (taxYear: Int) => s"Payments into pensions for 6 April ${taxYear - 1} to 5 April $taxYear"
-    val expectedTitle = "Relief at source (RAS) pensions"
-    val expectedErrorTitle = s"Error: $expectedTitle"
-    val expectedHeading = "Relief at source (RAS) pensions"
-    val expectedButtonText = "Continue"
-    val yesText = "Yes"
-    val noText = "No"
-  }
-
-  object CommonExpectedCY extends CommonExpectedResults {
-    val expectedCaption: Int => String = (taxYear: Int) => s"Payments into pensions for 6 April ${taxYear - 1} to 5 April $taxYear"
-    val expectedTitle = "Relief at source (RAS) pensions"
-    val expectedErrorTitle = s"Error: $expectedTitle"
-    val expectedHeading = "Relief at source (RAS) pensions"
-    val expectedButtonText = "Continue"
-    val yesText = "Yes"
-    val noText = "No"
-  }
-
-  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = Seq(
-    UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
-    UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
-    UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
-    UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY))
-  )
+  val userScenarios: Seq[UserScenario[_, _]] = Seq.empty
 
   ".show" should {
```

Comment:
It looks like two scenarios were dropped. Why was that?

Reply:
`messages.cy` is currently in english. We haven't done the translations

Reply:
Does this response belong to a different comment? :)

Reply:
Confirmed that the two scenarios were restored.